guides/common/modules/proc_booting-virtual-machines.adoc

```diff
@@ -48,19 +48,19 @@ If you want to change the default values in the template, clone the template and
 . Set *PXE Loader* to *iPXE Embedded*.
 . Select the *Templates* tab.
 . From the *iPXE template* list, select *Review* to verify that the *Kickstart default iPXE* template is the correct template.
-. Configure the `dhcpd.conf` file as follows:
-+
+
+On {ProjectServer}, run:
 [options="nowrap" subs="+quotes,attributes"]
 ----
-if exists user-class and option user-class = "iPXE" {
-  filename "http://{smartproxy-example-com}:8000/unattended/iPXE?bootstrap=1";
-} # elseif existing statements if non-iPXE environment should be preserved
+{foreman-installer} --foreman-proxy-dhcp-ipxefilename "http://{foreman-example-com}/unattended/iPXE?bootstrap=1"
 ----
-+
-If you use an isolated network, use a {SmartProxyServer} URL with TCP port `8000`, instead of the URL of {ProjectServer}.
-+
+
 [NOTE]
 ====
-If you have changed the port using the `--foreman-proxy-http-port installer` option, use your custom port.
-You must update the `/etc/dhcp/dhcpd.conf` file after every upgrade.
+Ensure that `--foreman-trusted-proxies` is configured correctly on the {ProjectServer}.
 ====
+
+[options="nowrap" subs="+quotes,attributes"]
```

Comment:
I thought it was implied that if the paragraph belonging to the command was removed, the command itself was also redundant. Please drop this too (and in the other file too).

Reply:
Addressed :+1: